library/actions/push.go

```diff
@@ -8,14 +8,18 @@ import "github.com/go-vela/types/constants"
 // Push is the library representation of the various actions associated
 // with the push event webhook from the SCM.
 type Push struct {
-	Branch *bool `json:"branch"`
-	Tag    *bool `json:"tag"`
+	Branch       *bool `json:"branch"`
+	Tag          *bool `json:"tag"`
+	DeleteBranch *bool `json:"delete:branch"`
+	DeleteTag    *bool `json:"delete:tag"`
```

Comment:
i didn't check closely, but i don't think we use `:` in any other JSON keys. I'd propose either `delete_*` or `*_delete`

Reply:
yeah I looked around and it is main `delete_*` format so I updated that.